packages/sit-onyx/src/components/OnyxDataGrid/features/resizing/resizing.ts

```diff
@@ -99,6 +110,10 @@ export const useResizing = createFeature(
           thAttributes: mergeVueProps(thAttributes, column.thAttributes),
         };
       });
+
+      return showLastCol.value
+        ? [...columns, { key: EMPTY_COLUMN, type: EMPTY_COLUMN, label: "" }]
+        : [...columns];
```

Comment:
```suggestion
     if (showLastCol.value) columns.push({ key: EMPTY_COLUMN, type: EMPTY_COLUMN, label: "" });
     return columns;
```

`...columns` creates a copy of the columns in memory so we can optimise the memory usage here a littlebit by reusing the existing array :)